### DISCUSSION TITLE
Fragen zur Pylontech Batterie

### DISCUSSION THREAD

**Discussion by swingstate** [hoylabs/OpenDTU-OnBattery#401] [General]
Da ja einige von euch diese Batterie nutzen - 

Lassen sich die zwei Griffen an den äußeren rechten/linken Seiten der Batterie abschrauben bzw. wie sind diese befestigt?
Ich brauche diese nicht und ohne würde die Batterie leichter in meine Kiste auf der Terrasse passen.

**Comment by madmartin** [comment]
Ja, die "Ohren" lassen sich abschrauben.
Es ist ein gewinkeltes Blech,was beim US3000C mit sechs Senkkopfschrauben befestigt ist. Wenn man es abschraubt ist die Seitenfläche des Gehäuse glatt. 
Ich hab grad ins Manual gesehen, da gibts auch ein schönes Bild dazu.
Wenn die Batterie auf der Terasse steht mußt Du ein Auge auf die Temperatur der Batterie haben... laut Anleitung 0°C~ 50°C beim Aufladen und -10°C~ 50°C Entladen

![grafik](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/4208120/d2591bbf-e85f-44ea-b4d7-3b3bd99bc5d7)

**Reply by swingstate** [reply]
Vielen Dank Martin! 
Ich beheize meine Batterien, somit sollte es da kein Problem geben.
